.github/workflows/release.yml

```diff
@@ -117,15 +117,24 @@ jobs:
           shared-key: release-${{ matrix.target }}
           cache-on-failure: true
 
+      # On Windows, Lassie build script copies golassie.dll to the target directory. When
+      # Swatinem/rust-cache loads cached data, it does not restore that DLL file. To fix the
+      # problem, we remove the cached Lassie version to force `cargo build` to re-run the build
+      # script, which will copy `golassie.dll`.
+      - name: DEBUGGING | Force rebuild of Lassie
```

Comment:
Great question! 

These DEBUGGING steps are executed only when we run the workflow from a pull request. By default, we run the workflow for new releases only, and these steps are skipped.

When modifying the release script, the first thing I do is to change the list of triggers to run the workflow for all pull requests too - see [760e729](https://github.com/filecoin-station/zinnia/pull/248/commits/760e72919d7109ecc8101472d79e553e0aa9a4b3). Only with this change in place, we will start running DEBUGGING steps.

What would be a good place where to capture this information? In the workflow file?

Reply:
I see! That's a useful pattern 👏 

Since you need to modify the workflow anyway, what do you think about commenting out the `DEBUG` steps, and adding a comment next to the commented out debug trigger?

Reply:
Or maybe we can introduce a debug variable, so we don't need to work with comments?

Reply:
👉🏻 https://github.com/filecoin-station/zinnia/pull/251